sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb

```diff
@@ -75,6 +75,10 @@
     "region = session.boto_region_name\n",
```

Comment:
<p>You're actually downloading from our S3 bucket....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>This was not changed by me and I think it was standard practice to have the datasets in our bucket to avoid errors downloading.</p>